articles/native-platforms/android/index.yml

```diff
@@ -15,3 +15,4 @@ snippets:
 seo_alias: android
 articles:
   - 01-login
+  - 07-authorization
```

Comment:
@tenmaster 
This does not agree with the index.yml of this PR: https://github.com/auth0/docs/pull/1778/files

Also, we were missing the `02-custom-login` which I added at the `yml` file earlier, so you need to rebase in order for me to merge this.